cg/constants/priority.py

```diff
@@ -41,3 +43,8 @@ class Priority(IntEnum):
     Priority.express: SlurmQos.EXPRESS,
     Priority.clinical_trials: SlurmQos.NORMAL,
 }
+
+
+class WorkflowManager(StrEnum):
```

Comment:
Should we add this here instead: `cg/constants/constants.py`. It is more a general cg feature than a priority concept